Wellness Plot

drippydhruv opened the pull request:
CHANGES: Added to the plot-handler.js file to include the API to plot a requested wellness metric and the date range. Added conditionals to the handle selector which will determine based on url to return either getExercisePlot or getWellnessPlot. Also changed name of constructors for exercise plot so it is not confused with the wellness constuctors. 

TESTING:  tested using postman with own database. Populated database with wellness entries and tried different requests in the URL with different wellness metrics and date ranges. 

LINK OF ISSUE: https://github.com/PietropaoloV/Atlas/issues/84

PietropaoloV reviewed (changes requested):
Make sure you add unit tests for your code in the plotspec.js. Also update any changes in the spec file with the changes made to my code. We don't want the unit tests breaking because you made changes. Please confirm all test are running and passing using NPM test

drippydhruv commented:
I have edited the same PR as the initial Wellness Plot one I submitted yesterday. 
ORIGINAL CHANGES: Added to the plot-handler.js file to include the API to plot a requested wellness metric and the date range. Added conditionals to the handle selector which will determine based on url to return either getExercisePlot or getWellnessPlot. Also changed name of constructors for exercise plot so it is not confused with the wellness constuctors. 

NEW CHANGES:  Added a different filtering method for getWellnessPlot. I added testing in the plot-handler-spec.js file for getWellnessPlot method. I created a new file for generating wellness entries(wellness-testing-util.js). 

TESTING: tested using postman with own database. Populated database with wellness entries and tried different requests in the URL with different wellness metrics and date ranges.

NEW TESTING: tested the unit testing with jasmine npm, all checks passed with the proper expected codes and outputs. 

LINK OF ISSUE: https://github.com/PietropaoloV/Atlas/issues/84
